src/desktop/apps/about/stylesheets/index.styl

```diff
@@ -14,6 +14,8 @@ header-margin = 180px
     margin-top (header-margin / 1.5)
   &:last-child
     margin-bottom (header-margin / 1.5)
+    > div:last-child
+      width calc(100% \- 400px)
```

Comment:
Whoa, I've not seen this `\-` thing before - what does it do??

Reply:
@jonallured probably this makes the minus work as it should, just like `\` inside backticks, you know?